src/foam/u2/crunch/CrunchController.js

```diff
@@ -364,7 +365,10 @@ foam.CLASS({
           await targetDAO.put(capable);
         });
       }
-
+      for ( let i = 0; i < x.wizardlets.length; i++ ) {
```

Comment:
```suggestion
      // TODO: choose better location for this or ensure it's in context
      for ( let i = 0; i < x.wizardlets.length; i++ ) {
```

Reply:
not to fix right now - just want to add this `// todo` comment because this `wizardController` property is something I'd like to fix